index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/styles.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="header-container">
+    <div class="header-item header-item1"><time id="time">9:41</time></div>
+    <div class="header-item header-item2"><img id="signal" src="src/signal.png" alt="signal"></div>
+    <div class="header-item header-item3"><p id="name">Anand Pathak</p></div>
+    <div class="header-item header-item4"><img id="tab" src="src/tab.png" alt="tab"></div>
+  </header>
+
+  <section class="intro-section-container">
+    <p class="intro-section-item" id="intro-text1">Hey There. I'm Anand</p>
```

Comment:
I suggest you change this to a `<h1>` tag. This is the maintitle of your page. Its good practice use the semantically correct tags If you want that google (and other search engines) knows about a your page.  [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.